### ISSUE TITLE
NVDA doesn't announce the document or window title when Microsoft Office was first opened.

### ISSUE THREAD

**Issue by supanut2000** [nvaccess/nvda#5768]
When I open any Microsoft Office apps, NVDA won't say the window or document title. This issue affects Office 2013 and 2016, but not Office 2010. As a result NVDA fails to read the contents of the document, unless if I switch to another app and then back.

**Comment by fernando-jose-silva** [comment]
Good morning, my dears.
 This problem also occurs with me, trying to help follows a log that shows what happens after pressing enter on a file in Word.
[office.txt](https://github.com/nvaccess/nvda/files/140717/office.txt)

**Comment by supanut2000** [comment]
Here's the log file that I managed to create this morning from my Surface 3.
[nvda2.txt](https://github.com/nvaccess/nvda/files/146149/nvda2.txt)

**Comment by ehollig** [comment]
This sounds related to #5699

**Comment by Adriani90** [comment]
@supanut2000, @fernando-jose-silva are you still able to reproduce this issue? Could you please enable UIA in MS Word  in last NVDA Alpha Version and see if the isue still occurs? at least for me I cannot reproduce this issue in MS Word 2016 with UIA enabled. Thank you.

**Comment by fernando-jose-silva** [comment]
On a machine with windows 10 17763.292
Microsoft Office version 16.0.11126.20266
nvda alfa 16682
I can not reproduce this problem any more.
I've heard of users of older versions of windows 10 build 16299 nvda 2018.4.1 and office 16.0.4xxx that does not have the option of uia for word that this problem still occurs, but at the moment I do not have access to these machines to get the log.

**Comment by Adriani90** [comment]
Closing this issue in favor of #8389.
